docs/janssen-server/recipes/passkey-impl-guide.md

```
@@ -0,0 +1,144 @@
+---
+
+## Admin Operations
+
+System administrators can monitor and manage passkeys using:
+
+### Text-Based UI (TUI)
+* Navigate to the **Users** section.
+* Search for a user inum.
+* Manage, inspect, or delete FIDO2 registration entries associated with the user profile.
+
+### Command Line (Jans CLI)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)